src/core/abr/representation_chooser.ts

```diff
@@ -398,7 +408,12 @@ export default class RepresentationChooser {
 
       // -- AUTO mode --
       let inStarvationMode = false; // == buffer gap too low == panic mode
-      return observableCombineLatest(clock$, maxAutoBitrate$, deviceEvents$)
+      return observableCombineLatest(
+        clock$, // playback situation evolves
+        maxAutoBitrate$, // the maximum auto bitrate is updated
+        deviceEvents$, // device situation changes
+        this._reEstimate$.pipe(startWith(undefined)) // new estimates have come
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this redundant "undefined". ([typescript:S4623](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4623))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=336&issues=AWaC9RiAn3WdA7VnLu3c&open=AWaC9RiAn3WdA7VnLu3c)

[ISSUE_KEY]: AWaC9RiAn3WdA7VnLu3c